test/log/log.go

```diff
@@ -0,0 +1,23 @@
+package log
```

Comment:
This package might be redundant. I added it just because ```NewHelper()``` function is asking for logger

Reply:
I don't have strong opinions, we can keep it.